job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/DslScriptLoader.groovy

```diff
@@ -118,18 +162,6 @@ class DslScriptLoader {
         idx > -1 ? fileName[0..idx - 1] : fileName
     }
 
-    /**
```

Comment:
It's good that `DslScriptLoader` no longer relies on static methods, but do not move any methods. Leave these where they are and mark them as deprecated. Then we can remove them eventually.